### DISCUSSION TITLE
Why is it displayed that it can't bind the port

### DISCUSSION THREAD

**Discussion by Guo-Chenxu** [etcd-io/etcd#16469] [Q&A]
Maybe it seems to be too fool, but after STFW and RTFM I still don't know why I have this error and how I can solve it.

I have run `etcd` as an administer, opening firewall, and there is not any service occupying port 2380

the error is in this picture:
<img src="https://cdn.jsdelivr.net/gh/Guo-Chenxu/imgs@main/imgs/202308241901013.png"/>

my env:
```
etcd Version: 3.5.9
Git SHA: bdbbde998
Go Version: go1.19.9
Go OS/Arch: windows/amd64
```

Appreciate!!!

**Comment by Guo-Chenxu** [comment]
I don't know why, but it revives after restarting the pc.

**Reply by jmhbnz** [reply]
Hey @Guo-Chenxu - Can I please confirm this issue is now resolved following the restart and is no longer occurring?

**Reply by Guo-Chenxu** [reply]
maybe, at least for now, it is healthy
